lib/game.rb

```diff
@@ -0,0 +1,43 @@
+class Game
+
+  PLAYER_ONE_MARK = "X"
+  PLAYER_TWO_MARK = "O"
+
+  def initialize(display, board)
+    @display = display
+    @board = board
+  end
+
+  public
+
+  def play
+    welcome_players
+    present_board
+    player_turns
+  end
+
+  private
+
+  def welcome_players
+    @display.welcome
+  end
+
+  def present_board
+    @display.present_board(@board.grid)
+  end
+
+  def player_turns
+    one_turn(PLAYER_ONE_MARK)
+    until @board.win?
+      one_turn(PLAYER_TWO_MARK)
+      player_turns
```

Comment:
A recursive call inside a loop is pretty mind-bending! Think about how you could simplify this 😺